apps/backoffice-v2/src/lib/blocks/hooks/useKYCBusinessInformationBlock/useKYCBusinessInformationBlock.ts

```diff
@@ -0,0 +1,7 @@
+import { useManualReviewBlocksLogic } from '@/lib/blocks/variants/ManualReviewBlocks/hooks/useManualReviewBlocksLogic/useManualReviewBlocksLogic';
+
+export const useKYCBusinessInformationBlock = () => {
+  const { businessInformationBlock } = useManualReviewBlocksLogic();
+
+  return businessInformationBlock;
+};
```

Comment:
Use `useEntityInfoBlock` directly. If you don't mind please also rename it to `useCaseInfoBlock`. 🙏🏼